spec/lib/bamboozled/api/base_spec.rb

```diff
@@ -15,4 +15,16 @@
     base = Bamboozled::API::Base.new("x", "x", { log_format: :curl })
     base.send(:request, :get, "test")
   end
+
+  context 'when changes API BASE url' do
```

Comment:
`  context 'when changing the base API url' do`
or maybe
`  context "when changing the base API url" do`

The style the repo uses is double quotes everywhere, maybe we should be consistent